poc/vdaf_poplar1.sage

```diff
@@ -34,7 +34,7 @@ class Poplar1(Vdaf):
     AggResult = Vec[Unsigned]
 
     @classmethod
-    def measurement_to_input_shares(Poplar1, measurement):
+    def measurement_to_input_shares(Poplar1, measurement, nonce):
```

Comment:
nit: In Python (i.e., Sage :D) it's conventional to prefix ignored parameters with an underscore.
```suggestion
    def measurement_to_input_shares(Poplar1, measurement, _nonce):
```